docs/pages/guides/authentication.md

```diff
@@ -1128,11 +1128,11 @@ export default function App() {
 - Replace `YOUR_GUID` with your "Web client ID" and open this link:
   - https://console.developers.google.com/apis/credentials/oauthclient/YOUR_GUID.apps.googleusercontent.com
 - Under "URIs" add your hosts URLs
-  - Web dev: https://localhost:19006
+  - Web dev: http://localhost:19006
   - Expo Client Proxy: https://auth.expo.io
 - Under "Authorized redirect URIs"
-  - Web dev: https://localhost:19006 -- this is assuming you want to invoke `WebBrowser.maybeCompleteAuthSession();` from the root URL of your app.
-  - Expo Client Proxy: https://auth.expo.io/@yourname/your-app
+  - Web dev: http://localhost:19006 -- this is assuming you want to invoke `WebBrowser.maybeCompleteAuthSession();` from the root URL of your app.
```

Comment:
I don't understand? This is supposed to be https because auth-session uses crypto API which requires a secure origin.